docs/modules/ROOT/pages/how-to-guides/Secure-your-supply-chain/proc_java_dependencies.adoc

```diff
@@ -132,7 +208,7 @@ kubectl label secret jvm-build-service-upload spi.appstudio.redhat.com/upload-se
 
 Once these steps are completed the secret should disappear, and the system is ready to use.
 
-=== Examining the System State
+== Examining the System State
```

Comment:
For titles, you only capitalize the first word and formal nouns:

== Examining the system state